presto-main/src/test/java/com/facebook/presto/sql/planner/planPrinter/TestPlanPrinter.java

```diff
@@ -186,4 +191,37 @@ public void testDomainTextFormatting()
                 Domain.onlyNull(VARCHAR),
                 "[NULL]");
     }
+
+    @Test
+    public void testPrintNativeExecutionNode()
+    {
+        ImmutableMap<VariableReferenceExpression, RowExpression> map = ImmutableMap.of(
+                COLUMN_VARIABLE,
+                COLUMN_VARIABLE);
+        TableScanNode scan = PLAN_BUILDER.tableScan(
+                TABLE_HANDLE_WITH_LAYOUT,
+                ImmutableList.of(COLUMN_VARIABLE),
+                ImmutableMap.of(COLUMN_VARIABLE, COLUMN_HANDLE));
+        LimitNode limit = PLAN_BUILDER.limit(10, scan);
+        ProjectNode project = PLAN_BUILDER.project(new Assignments(map), limit);
+        NativeExecutionNode nativeExecution = PLAN_BUILDER.nativeExecution(project);
+
+        PlanFragment testFragment = new PlanFragment(
+                new PlanFragmentId(0),
+                nativeExecution,
+                ImmutableSet.of(),
+                SOURCE_DISTRIBUTION,
+                ImmutableList.of(nativeExecution.getId()),
+                new PartitioningScheme(Partitioning.create(SOURCE_DISTRIBUTION, ImmutableList.of()), ImmutableList.of()),
+                StageExecutionDescriptor.ungroupedExecution(),
+                false,
+                StatsAndCosts.empty(),
+                Optional.empty());
+
+        String textPlan = PlanPrinter.textPlanFragment(testFragment, FUNCTION_AND_TYPE_MANAGER, TEST_SESSION, true);
+        assertTrue(textPlan.contains("NativeExecution"));
+        assertTrue(textPlan.contains("Project"));
+        assertTrue(textPlan.contains("Limit[10]"));
+        assertTrue(textPlan.contains("TableScan"));
```

Comment:
Can we have more detailed assertion like the test above? We can have longer string  (regex) matching
```
    - NativeExecution => [column:varchar]
        - Project[projectLocality = UNKNOWN] => [column:varchar]
            - Limit[10] => [column:varchar]
                - TableScan[TableHandle {connectorId='testConnector', connectorHandle='com.facebook.presto.testing.TestingMetadata$TestingTableHandle@1af56f7', layout='Optional[INSTANCE]'}, grouped = false] => [column:varchar]
                        LAYOUT: INSTANCE
                        column := com.facebook.presto.testing.TestingMetadata$TestingColumnHandle@dad7ca35